contracts/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ project(contracts)
 set(EOSIO_WASM_OLD_BEHAVIOR "Off")
 find_package(eosio.cdt)
 
-set(ICON_BASE_URL "http://127.0.0.1/ricardian_assets/eosio.contracts/icons")
+set(ICON_BASE_URL "https://wax.io/ricardian_assets/eosio.contracts/icons")
```

Comment:
I am confused here. We have the icons stored in https://github.com/worldwide-asset-exchange/wax-system-contracts/tree/develop/contracts/icons and the suggested address is not redirecting to it. Is it the idea, to use wax.io to redirect to the current/working icon directory?

Reply:
yeah let's punt this for now